lib/smart_answer/calculators/marriage_abroad_calculator.rb

```diff
@@ -260,6 +261,10 @@ def document_download_link_if_opposite_sex_resident_of_uk_countries?
       MarriageAbroadDataQuery::NO_DOCUMENT_DOWNLOAD_LINK_IF_OS_RESIDENT_OF_UK_COUNTRIES.exclude?(ceremony_country)
     end
 
+    def consular_fee(service)
+      @rates_query.rates[service]
```

Comment:
It might be a good idea to raise an exception here if the `service` key doesn't exist in `@rates_query.rates`. Having said that, I think we already have this same problem in other code that uses `RatesQuery` so I think it could also be changed outside of this PR.